src/actions/accountsActions.js

```diff
@@ -76,7 +89,7 @@ export const addAccountAction = (
 
     if (existingAccount) {
       // $FlowFixMe: flow gets confused here
-      updatedAccounts.push({ ...existingAccount, extra: accountExtra });
+      updatedAccounts.push({ ...existingAccount, extra: parsedAccountExtra });
```

Comment:
Try remove the above flow fix me, could work now... maybe... ;-)

Reply:
Still bites 👿